manifests/vsphere-67u3/vanilla/deploy/vsphere-csi-controller-deployment.yaml

```diff
@@ -22,30 +26,30 @@ spec:
         node-role.kubernetes.io/master: ""
       tolerations:
         - operator: "Exists"
-          key: node-role.kubernetes.io/master
           effect: NoSchedule
+        - operator: "Exists"
+          effect: NoExecute
       dnsPolicy: "Default"
       containers:
         - name: csi-attacher
-          image: quay.io/k8scsi/csi-attacher:v1.1.1
+          image: quay.io/k8scsi/csi-attacher:v2.0.0
           args:
             - "--v=4"
             - "--timeout=300s"
             - "--csi-address=$(ADDRESS)"
+            - "--leader-election"
           env:
             - name: ADDRESS
               value: /csi/csi.sock
           volumeMounts:
             - mountPath: /csi
               name: socket-dir
         - name: vsphere-csi-controller
-          image: gcr.io/cloud-provider-vsphere/csi/release/driver:v1.0.2
+          image: vmware/vsphere-csi:<vsphere_csi_ver>
```

Comment:
What is the plan with `<vsphere_csi_ver>`? This should be addressed in upcoming PR, cannot be used otherwise.